examples/enclave_manager/tcf_enclave_manager/enclave_manager.py

```diff
@@ -225,6 +230,8 @@ def create_enclave_signup_data():
     return enclave_signup_data
 
 # -----------------------------------------------------------------
+
+
 def execute_work_order(enclave_data, input_json_str, indent=4):
     """
     Submits workorder reqeust to Worker enclave and retrieves the response 
```

Comment:
s/reqeust/request/

Reply:
Sorry, I don't get it. Would you please explain your point in detail? @danintel

Reply:
I guess he mean to fix spelling errors. Search and Replace reqeust/request/.

Reply:
Gotcha ~